gen3-integration-tests/services/audit.py

```diff
@@ -36,11 +37,15 @@ def checkQueryResults(self, logCategory, user, params, expectedResults):
             data = response.json()
             # Counter to check response is recieved within 5 mins
             if len(data["data"]) != 0:
-                logger.info(data["data"])
-                for key, val in expectedResults.items():
-                    # Get the first entry of json data
-                    assert data["data"][0][key] == expectedResults[key]
-                return True
+                # Convert json to dataframe for easy querying of records using username
```

Comment:
It seems audit service provides a mechanism to query by username based on configuration - https://github.com/uc-cdis/audit-service/blob/master/src/audit/routes/query.py#L132, let's ask if we can use that instead of explicitly filtering it in the test code, that might be faster. If not, let's get this merged.